Dockerfile

```diff
@@ -19,6 +19,9 @@ RUN yum install -y fontconfig freetype freetype-devel fontconfig-devel libstdc++
   tar -xjvf ./phantomjs-2.1.1-linux-x86_64.tar.bz2 --strip-components 1 -C /opt/phantomjs/ && \
   ln -s /opt/phantomjs/bin/phantomjs /usr/bin/phantomjs
 
+RUN curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > /cc-test-reporter
+RUN chmod +x /cc-test-reporter
+
```

Comment:
I couldn't get the Dockerfile to build locally with `docker-compose up` on this branch, but it failed on the `groupadd -r gibct` command with code `139`  which seems odd

Reply:
It's likely unrelated to this branch.  https://github.com/department-of-veterans-affairs/gibct-data-service/pull/431 just changed the Dockerfile significantly.